DataTool/ToolLogic/Extract/ExtractMusic.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.IO;
+using DataTool.Flag;
+using static DataTool.Program;
+using DataTool.SaveLogic;
+using TankLib.STU.Types;
+using static DataTool.Helper.STUHelper;
+using System.Collections.Generic;
+
+namespace DataTool.ToolLogic.Extract.Debug
+{
+    [Tool("extract-music", Description = "Extracts sound files which are identified as music.", CustomFlags = typeof(ExtractFlags), IsSensitive = true)]
+
+    public class ExtractMusic : ITool
+    {
+
+        Dictionary<UInt32, string> music_types = new Dictionary<uint, string>
+        {   { 3851462253, "LoadingScreen" },
+            { 3043319348, "Retribution" },
+            { 935766596, "Uprising"},
+            { 1935561723, "StormRising1" },
+            { 3069540790, "StormRising2"},
+            { 1287967541, "Junkenstein"},
+            { 3187645923, "PvP" },
+            { 3991942870, "Stinger" },
+            { 397648059,  "PvE"},
+            { 2930411284, "MainMenuTheme"},
+            { 251430280, "RoundNearEnd"}
```

Comment:
What is PVP, PVE and Stinger type music?

Reply:
PvP is for music which can be used on any map in PvP (payload near destination, we are running out of time e.t.c.), same for pve. 
A stinger is a short clip of music. If name is confusing then i can change it.